index.html

```diff
@@ -1,9 +1,157 @@
+      alt="girl on a picnic"
+    />
+    <hr />
+    <ol>
+      <li>Знакомство с Web</li>
+      <ul>
+        <li>Краткая история Web и что такое интернет.</li>
+        <li>Как устроен браузер и как он получает страницы по сети.</li>
+        <li>Что такое Git?</li>
+        <li>Создание первого репозитория.</li>
+        <li>о файле .gitignore</li>
+      </ul>
+      <li>HTML-вёрстка&#8212;первые шаги</li>
+      <ul>
+        <li>Настройка рабочей среды.</li>
+        <li>Краткая история HTML.</li>
+        <li>Какие бывают теги и что такое атрибут.</li>
+        <li>Тэги для работы с текстом.</li>
+        <li>Создание собственной страницы.</li>
+      </ul>
+      <li>Погружение в HTML-разметку</li>
+      <ul>
+        <li>Мета-теги.</li>
+        <li>Мнемоника.</li>
+        <li>Тег для работы с изображениями.</li>
+        <li>Теги для работы со списками.</li>
+        <li>Теги для работы с таблицами.</li>
+        <li>Теги для работы с формами.</li>
+        <li>Div-блоки.</li>
+      </ul>
+    </ol>
+    <hr />
+    <table border="1" width="30%">
+      <tr>
+        <th>Tag</th>
+        <th>Description</th>
+      </tr>
+      <tr>
+        <td>&#60;table&#62;</td>
+        <td>Defines a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tr&#62;</td>
+        <td>Defines a row in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;th&#62;</td>
+        <td rowspan="2">Defines a "header cell"/"cell" in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;td&#62;</td>
+      </tr>
+      <tr>
+        <td colspan="2">Rarely used</td>
+      </tr>
+      <tr>
+        <td>&#60;caption&#62;</td>
+        <td>Defines a table caption</td>
+      </tr>
+      <tr>
+        <td>&#60;thead&#62;</td>
+        <td>Groups the header content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tbody&#62;</td>
+        <td>Groups the body content in a table</td>
+      </tr>
+      <tr>
+        <td>&#60;tfoot&#62;</td>
+        <td>Groups the footer content in a table</td>
+      </tr>
+    </table>
+    <hr />
+    <form>
+      <fieldset>
+        <legend>My first form</legend>
+        <p>
+          <label for="name">Name</label><br />
+          <input
+            type="text"
+            id="name"
+            name="Name"
+            placeholder="Name"
+            required
+          /><br />
+          <label for="email">Email</label><br />
+          <input type="email" id="email" name="Email" required /><br />
+          <label for="password">Password</label><br />
+          <input type="password" id="password" name="password" required /><br />
+        </p>
+        <p>
+          <label for="level">Your programming level</label><br />
+          <input type="range" id="level" name="level" min="0" max="10" />
+        </p>
+        <p>
+          What does HTML stand for?<br />
+          <input type="checkbox" id="hyperlinks" name="hyperlinks" />
```

Comment:
please add value
[read more about value](https://webref.ru/html/button/value#:~:text=%D0%9E%D0%BF%D1%80%D0%B5%D0%B4%D0%B5%D0%BB%D1%8F%D0%B5%D1%82%20%D0%B7%D0%BD%D0%B0%D1%87%D0%B5%D0%BD%D0%B8%D0%B5%20%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B8%2C%20%D0%BA%D0%BE%D1%82%D0%BE%D1%80%D0%BE%D0%B5%20%D0%B1%D1%83%D0%B4%D0%B5%D1%82,%D0%BA%D0%BD%D0%BE%D0%BF%D0%BA%D0%B5%2C%20%D1%82%D0%B0%D0%BA%20%D0%B1%D1%8B%D1%82%D1%8C%20%D0%B8%20%D1%81%D0%B0%D0%BC%D0%BE%D1%81%D1%82%D0%BE%D1%8F%D1%82%D0%B5%D0%BB%D1%8C%D0%BD%D1%8B%D0%BC.)